src/main/java/nl/novi/Les10SpringBootTechItEasyApplication/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,46 @@
+package nl.novi.Les10SpringBootTechItEasyApplication.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TelevisionsController {
+    @GetMapping("/televisions")
+    public ResponseEntity<Object> getTelevisions() {
+        return ResponseEntity.ok();
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<Object> getTelevisions(@PathVariable int id) {
+//        if(id>3) {
+//            throw new RecordNotFoundException("id > 3");
+//        }
+        return ResponseEntity.ok();
+    }
+
+    @PostMapping("/televisions")
+    public ResponseEntity<Object> postTelevision(@RequestBody String televisionName) {
+        return ResponseEntity.created();
+    }
+
+    @PutMapping("/television/{id}")
```

Comment:
Waarom gebruik je hier "/television"  terwijl je hiervoor "/television" gebruikte? Probeer hier consequent in te zijn. Een uniforme API is een van de requirements voor een goede REST applicatie.

Reply:
dat is een fout van mij. DIe heb ik in mijn code inmiddels aangepast